docs/authentication.md

```diff
@@ -0,0 +1,130 @@
+    * Auditor-side properties
+      * `scalar.dl.auditor.authentication.method` (set "digital-signature")
+* Ledger authentication 
+    * Ledger-side properties
+        * `scalar.dl.ledger.proof.enabled` (set true)
+            * It is required because Ledger authentication uses the signatures of Asset Proofs.
+        * `scalar.dl.ledger.proof.private_key_pem` or `scalar.dl.ledger.proof.private_key_path`  
+            * It is used for signing Asset Proofs.
+    * Auditor-side properties
+        * `scalar.dl.auditor.ledger.cert_holder_id`
+        * `scalar.dl.auditor.ledger.cert_version`
+            * It is used for verifying the signatures of Asset Proofs.
+* Auditor authentication
+    * Ledger-side properties
+        * `scalar.dl.ledger.auditor.cert_holder_id`
+          * It is used for verifying an Auditor-generated signature attached to a client request.
+        * `scalar.dl.ledger.auditor.cert_version`
+          * It is used for verifying an Auditor-generated signature attached to a client request.
+    * Auditor-side properties
+        * `scalar.dl.auditor.cert_holder_id`
+          * It is used for calling Ledger services directly using the client library.
+        * `scalar.dl.auditor.cert_version`
+          * It is used for calling Ledger services directly using the client library.
+        * `scalar.dl.auditor.private_key_pem` or `scalar.dl.auditor.private_key_path`
+          * It is used for signing a client request and a request from Auditor to Ledger.
+
+### HMAC
+
+* Client authentication
+    * Client-side properties
+        * `scalar.dl.client.authentication_method` (set "hmac")
+        * `scalar.dl.client.entity.id`
+            * It is used for identifying a client.
+        * `scalar.dl.client.entity.identity.hmac.secret_key`
+            * It is used for signing a request.
+        * `scalar.dl.client.entity.identity.hmac.key_version`
+    * Ledger-side properties
+      * `scalar.dl.ledger.authentication.method` (set "hmac")
+    * Auditor-side properties
+      * `scalar.dl.auditor.authentication.method` (set "hmac")
+
+* Ledger and Auditor authentication
+    * Ledger-side properties
+        * `scalar.dl.ledger.proof.enabled` (set true)
+            * It is required because Ledger authentication uses the signatures of Asset Proofs.
+        * `scalar.dl.ledger.authentication.hmac.cipher_key`
+            * It is used for signing and verifying messages/requests between Ledger and Auditor.
+    * Auditor-side properties
+        * `scalar.dl.auditor.authentication.hmac.cipher_key`
+            * It is used for signing and verifying messages/requests between Ledger and Auditor. It must be the same key as `scalar.dl.ledger.authentication.hmac.cipher_key`.
+
+ ## Prepare before executing requests
+
+ After configuration, you must do the following to prepare for issuing execution requests.
+
+### Digital Signatures
+* Register clients' certificates to Ledger (and Auditor).
```

Comment:
Does `(and Auditor)` in parentheses depend on the user's environment? If so, can we change it to the following?:

```suggestion
* Register clients' certificates to Ledger (and Auditor, if enabled).
```